metadata-models/src/main/pegasus/com/linkedin/ml/metadata/IntendedUserType.pdl

```diff
@@ -0,0 +1,22 @@
+namespace com.linkedin.ml.metadata
+
+/*
+* Primary Intended User Types or User Categories
+*/
+enum IntendedUserType {
+
+    /*
+    * Developed for Enterprise Users
+    */
+    ENTERPRISE
+
+    /*
+    * Developed for Hobbyists
+    */
+    HOBBY
+
+    /*
+    * Developed for Entertainment Purposes
+    */
+    ENTERTAINMENT
```

Comment:
Are these 3 representing the possible intended user types? If the intended users are LinkedIn members, which should be selected?

Reply:
Yeah these are the intended user types to start with. LinkedIn members would fall under `ENTERPRISE` in my opinion.